contracts/RMRK/library/RMRKLib.sol

```diff
@@ -32,7 +32,7 @@ library RMRKLib {
         returns (uint256, bool)
     {
         uint256 length = A.length;
-        for (uint256 i = 0; i < length; ) {
+        for (uint256 i; i < length; ) {
```

Comment:
Is this change more gas effective?

Reply:
I think there shouldn't be any difference once compiled

Reply:
I didn't actually test it. I'm following this:
https://github.com/transmissions11/solcurity

Reply:
You could also review everything using Mythril: https://github.com/ConsenSys/mythril should give you a feeling about improvement pre- and post-update